spring-boot-project/spring-boot/src/main/java/org/springframework/boot/web/reactive/filter/OrderedHiddenHttpMethodFilter.java

```diff
@@ -32,8 +31,7 @@
 	/**
 	 * The default order is high to ensure the filter is applied before Spring Security.
 	 */
-	public static final int DEFAULT_ORDER = FilterRegistrationBean.REQUEST_WRAPPER_FILTER_MAX_ORDER
-			- 10000;
+	public static final int DEFAULT_ORDER = -10000;
```

Comment:
I can understand why you'd want to drop a reference to this class but I am not sure copy/pasting the default (I mean `0`) is something we want to do. Perhaps we need a shared context for this?

WDYT?

Reply:
@snicoll Thanks for the quick feedback! I thought the same thing but didn't find a good place to move them. So I considered adding a constant container class like `FilterOrders` but with only one member, I wasn't sure it's okay. Any suggestion?

Reply:
@bclozel that was added as part of #14008 - Does the constant has to be the same for reactive? If it's the same then we should probably need some concept that gathers them. What do you think?